src/types/activity.ts

```diff
@@ -1,18 +1,47 @@
 export interface ActivityPayload {
+  /** the activity's id */
+  id?: string;
+  /** the activity's name */
   name: string;
+  /** activity type */
   type: number;
+  /** stream url, is validated when type is 1 */
   url?: string;
+  /** unix timestamp of when the activity was added to the user's session */
   created_at: number;
+  /** unix timestamp of when the activity was added to the user's session */
   timestamps?: ActivityTimestamps;
+  /** the id of the song on Spotify */
+  sync_id?: string;
+  /** the platform the game is being played on ("desktop", "samsung", or "xbox") */
+  platform?: string;
+  /** application id for the game */
   application_id?: string;
+  /** what the player is currently doing */
   details?: string;
+  /** the user's current party status */
   state?: string;
+  /** the emoji used for a custom status */
   emoji?: ActivityEmoji;
```

Comment:
You forgot the `session_id` here
```suggestion
  emoji?: ActivityEmoji;
  /** the id of the game or Spotify session */
  session_id?: string;
  /** information for the current party of the player */
```